[2024-07-18 06:35:36,454] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2024-04-08T00:00:00+00:00 [queued]>
[2024-07-18 06:35:36,506] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2024-04-08T00:00:00+00:00 [queued]>
[2024-07-18 06:35:36,507] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:35:36,508] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2024-07-18 06:35:36,509] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:35:36,546] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): pull_task> on 2024-04-08T00:00:00+00:00
[2024-07-18 06:35:36,557] {standard_task_runner.py:52} INFO - Started process 3108 to run task
[2024-07-18 06:35:36,576] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_example_dag', 'pull_task', '2024-04-08T00:00:00+00:00', '--job-id', '558', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom.py', '--cfg-path', '/tmp/tmp9c5fv0b_', '--error-file', '/tmp/tmp1jhf4jxd']
[2024-07-18 06:35:36,601] {standard_task_runner.py:77} INFO - Job 558: Subtask pull_task
[2024-07-18 06:35:36,824] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_example_dag.pull_task 2024-04-08T00:00:00+00:00 [running]> on host a868c7bacc12
[2024-07-18 06:35:37,055] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_example_dag
AIRFLOW_CTX_TASK_ID=pull_task
AIRFLOW_CTX_EXECUTION_DATE=2024-04-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-08T00:00:00+00:00
[2024-07-18 06:35:37,081] {logging_mixin.py:104} INFO - Pulled message: Hello from push_function!
[2024-07-18 06:35:37,086] {python.py:118} INFO - Done. Returned value was: None
[2024-07-18 06:35:37,145] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=xcom_example_dag, task_id=pull_task, execution_date=20240408T000000, start_date=20240718T063536, end_date=20240718T063537
[2024-07-18 06:35:37,494] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-18 06:35:37,575] {local_task_job.py:146} INFO - Task exited with return code 0
